presentation/sampleNotes.tex

```diff
@@ -0,0 +1,15 @@
+% To build an accompanying notes file, comment out the first line of `sampleSlides.tex`
+% and build this file. Using the pdf viewer Skim, you can then view the notes file on your
+% own computer while the audience views your projected slides.
+% To do this,
+% Open both sampleSlides.pdf and sampleNotes.pdf, change displays to "Extended."
+% Move sampleSlides.pdf to the projected screen, and leave sampleNotes.pdf on your screen.
+% In sampleSlides.pdf, select `View > Presentation Options > Synchronized Notes Document`
+% Then select `View > Presentation` to full-screen your slides.
+% The notes slide should then move synchronously with your presentation.
+% NOTE: Be sure to include a note at the bottom of every slide, or your presentations will not
+% be the same length and this trick won't work so well.
+% Be sure to practice before your talk :)
+\documentclass{beamer}
+\setbeameroption{show only notes}
+\input{./sampleSlides.tex}
```

Comment:
Can you please submit a separate PR for your slides?  I would be open to adding sample beamer slides, but I also have been working on some myself.  If you create a separate PR I might try to merge it with what I have and get back to you on this.

Reply:
Certainly. I was a little silly for this request and mixed too many commits together; let me start fresh and submit the PRs separately.

Reply:
I'm not sure how to create a separate pull request for the slides; I reverted my previous changes and re-committed the slides separately. Now that this request is open, it seems to be absorbing any new commits I make.